client/question/services/FeedbackGeneratorService.js

```diff
@@ -204,7 +269,7 @@ tie.factory('FeedbackGeneratorService', [
         previousMessage = previousFeedback.getParagraphs()[0].getContent();
 
         if (previousFeedback.getFeedbackCategory() ===
-            currentFeedbackCategory) {
+          currentFeedbackCategory) {
```

Comment:
I think the previous indentation is preferable, actually. It makes it easier to tell that the next line is the start of the child scope, and it left-aligns with the '(' on the previous line (which is generally what to do when the contents of a parentheses span multiple lines).

Reply:
I think I changed this to address a linter error on indentation.